addons/web/static/src/js/core/utils.js

```diff
@@ -500,6 +500,18 @@ var utils = {
     isComponent: function (value) {
         return value.prototype instanceof owl.Component;
     },
+    /**
+     * Checks if a keyboard event concerns
+     * the numpad decimal separator key
+     * while the NumLock modifier state is active.
+     * 
+     * @param {KeyboardEvent} ev
+     * @returns {boolean}
+     */
+    isNumpadDecimalSeparatorKey(ev) {
+        return ev.getModifierState('NumLock')
+            && (ev.code !== undefined && ev.code === 'NumpadDecimal');
```

Comment:
Which suggestion would you prefer ?

This one uses `KeyboardEvent.getModifierState()` and thus would not support the `shift+numpad_dot`:
```suggestion
        return ev.getModifierState('NumLock') && ev.code === 'NumpadDecimal';
```


or this one complies too with the current API and would support the `shift+numpad_dot`
but I am afraid that some system region settings may interfere (see first row, second cell of [this table](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values#Numeric_keypad_keys)) :
```suggestion
        return ev.key === '.' && ev.code === 'NumpadDecimal';
```

or maybe this one, but `keyCode` attribute [is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode):
```suggestion
        return ev.keyCode === 190 && ev.code === 'NumpadDecimal';
```

Reply:
I think I prefer the second one (check on key and code).
A (little) lookaroo gives that there is an ISO for it https://www.iso.org/schema/isosts/v1.0/doc/n-cdf0.html
a decimal point is either comma or dot.